### PR TITLE
[docs] can use "[[GUIDE GuideName]]" to include formatted guide text

### PR DESCRIPTION
**Pull request by anjakefala**
- [] make sure it builds in CI

### CODE REVIEW

File: processdoc.py [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+        self.cls = cls
+        self.helpsheet = vd.HelpSheet()
+        self.helpsheet.ensureLoaded()
+
+    def __getattr__(self, k):
+        return self.__getitem__(k)
+
+    def __getitem__(self, k):
+        longname = k.replace('_', '-')
+        binding = self.helpsheet.revbinds.get(longname, [None])[0]
```
Comment (ajkerrigan) [inline processdoc.py:29]: I ran into an issue while writing a JSON guide and tried to [address it](https://github.com/saulpw/visidata/pull/2268/files#diff-2c3a790d477d55deb5ed1617c7c2802245ae9eb8c35f04bd7104063c0d8f2658L125-R125) in saulpw/visidata#2268 by defining a fallback string when commands have no keybindings. Not sure what the right/best way is to handle it, but I'm guessing any update would need to happen in both vd's guide.py and here?

```suggestion
        binding = self.helpsheet.revbinds.get(longname, [None])[0] or '<unbound>'
```
```

Comment (saulpw) [reply processdoc.py:29]: Yes, good point!  We'll get your PR reviewed here shortly and make sure they're in sync.

Comment (ajkerrigan) [reply processdoc.py:29]: Cool cool, no rush on that. And I can certainly split the unbound-command fix into a separate smaller/more-focused pr if that's useful.

Comment (anjakefala) [reply processdoc.py:29]: Oh, I would love it if you seperated it out! I don't want its merging to be blocked on reviewing the json text. =) That seems really useful.

Comment (ajkerrigan) [reply processdoc.py:29]: Sure thing - saulpw/visidata#2273

**Pull request merged**